static/js/nodes.js

```diff
@@ -9,27 +9,30 @@
     url += '?vhost=' + encodeURIComponent(vhost)
   }
   let updateTimer = null
-  let data = null
-
-  if (data === null) {
-    update(render)
-  }
 
   function update (cb) {
-    lavinmq.http.request('GET', url).then(function (response) {
-      data = response
-      render(response)
+    const statsPromise = lavinmq.http.request('GET', url)
+    Promise.all([
+      statsPromise
+    ]).then(function (responses) {
+      const statsData = responses[0]
```

Comment:
are you sure there will always be one item? and why is an array returned if there's only a single item?